bylaws/003-membership.md

```diff
@@ -98,9 +98,13 @@ will be assumed.
 
 If, in the judgement of PHP-FIG, a Voting Representative is acting
 inappropriately and to the detriment of PHP-FIG's ability to meet its
-objectives, a vote may be taken to request a replacement Voting Representative in
-accordance with the [Voting Protocol bylaw][voting] or to expel the Member
-Project where replacing a Voting Representative is not possible.
+objectives, a vote may be taken to request a replacement Voting Representative,
+or to expel the Member Project where replacing a Voting Representative is not
```

Comment:
I dunno I read it again today and it makes perfect sense to me. Works on my machine! 💃 

The first section is saying that voting members can be kicked out, and that leads to the member ship project getting the boot if there is nobody else. The second section being added explains that votes can be had to kick member projects directly.

What's the trouble?